packages/jsii-dotnet-generator/src/Amazon.JSII.Generator.UnitTests/AssemblyGeneratorTests.cs

```diff
@@ -29,6 +29,17 @@ static string GetProjectFilePath(string dotnetPackage, string dotnetAssembly)
             return $"{Path.Combine(GetPackageOutputRoot(dotnetPackage), dotnetAssembly)}.csproj";
         }
 
+        static string GetAnchorFilePath(string dotnetPackage, string dotnetNamespace)
+        {
+            string path = GetPackageOutputRoot(dotnetPackage);
+
+            foreach (string token in dotnetNamespace.Split('.')) {
+                path = Path.Combine(path, token);
+            }
+
+            return Path.Combine(path, "Internal", "DependencyResolution", "Anchor.cs");
+        }
+
         static string GetTypeFilePath(string dotnetPackage, string dotnetNamespace, string dotnetType)
```

Comment:
_ private (be obvious)

Reply:
I'm find with adding explicit access modifiers where they are currently implied, but currently the codebase omits all redundant access modifiers. If we change this, we should change it across the whole codebase.

Created #217 to track this.